### ISSUE TITLE
Abbreviation for months and exception handler

### ISSUE THREAD

**Issue by guyskk** [icholy/durationpy#8]
#### Q1: It seems use `mo` is better than `mm` for months.

- https://writingexplained.org/english-abbreviations/months
- https://www.quora.com/What-is-the-proper-way-to-abbreviate-the-word-month

I suggest support both `mm` and `mo`, and deprecate `mm` to avoid break changes.

#### Q2: When duration parse failed, it's better to raise custom exception instead of `Exception`.

**Comment by icholy** [comment]
It's a good suggestion, but it's not worth breaking people's code.

**Comment by icholy** [comment]
@guyskk I'm open to accepting a PR to add a custom exception.

**Comment by guyskk** [comment]
I create a PR #9 , and I found the code style is hard to follow, my editor linter report lots of warnings. Do you accepting code style clean up?

I vendor this library into one of my project, with many code style change(follow PEP8), you can check it at: https://github.com/guyskk/validr/blob/master/src/validr/_vendor/durationpy.py .
